starlite/routes.py

```diff
@@ -107,14 +108,25 @@ def create_handler_kwargs_model(self, route_handler: BaseRouteHandler) -> Kwargs
         """
         dependencies = route_handler.resolve_dependencies()
         signature_model = get_signature_model(route_handler)
+
         path_parameters = set()
         for param in self.path_parameters:
             param_name = param["name"]
             if param_name in path_parameters:
                 raise ImproperlyConfiguredException(f"Duplicate parameter '{param_name}' detected in '{self.path}'.")
             path_parameters.add(param_name)
+
+        layered_parameters: Dict[str, FieldInfo] = {}
+        for layer in route_handler.ownership_layers():
+            if getattr(layer, "parameters", None):
+                for key, field_info in cast(Dict[str, FieldInfo], layer.parameters).items():  # type: ignore
+                    layered_parameters[key] = field_info
```

Comment:
```python
layered_parameters: Dict[str, FieldInfo] = {}
for layer in route_handler.ownership_layers():
    layered_parameters.update(getattr(layer, "parameters", {})
```

Reply:
The getattr is because the handlers don't have the `parameters` attribute right?

Reply:
Yup